mod-source-record-manager-server/src/main/java/org/folio/verticle/JobExecutionProgressVerticle.java

```diff
@@ -239,6 +256,7 @@ private Future<Boolean> updateJobExecutionIfAllRecordsProcessed(String jobExecut
                             parentExecution.withStatus(JobExecution.Status.COMMITTED)
```

Comment:
@JavokhirAbdullayev, could you please check this variant of code
```suggestion
                                                    .compose(allChildrenCompleted -> allChildrenCompleted
                            ? jobExecutionService.updateJobExecutionWithSnapshotStatus(
                            parentExecution.withStatus(JobExecution.Status.COMMITTED)
                              .withUiStatus(JobExecution.UiStatus.RUNNING_COMPLETE)
                              .withCompletedDate(new Date()), params)
                          .onSuccess(v -> sendEventToBulkOps(parentExecution, params))
                            : Future.succeededFuture()
                        )
```